frontend/Nouns/EditIntakeField/EditIntakeTextField.tsx

```diff
@@ -0,0 +1,109 @@
+import { ModalActions } from "../../Styleguide/Modal";
+import { FormEvent, useState } from "react";
+import { FormField } from "../../Styleguide/FormField";
+import { FormFieldLabel } from "../../Styleguide/FormFieldLabel";
+import { Input } from "../../Styleguide/Input";
+import { IntakeFieldItem, IntakeItemType } from "../CreateEditIntakeForm";
+import { EditItemProps } from "../EditIntakeItem";
+import { Toggle } from "../../Styleguide/Toggle";
+import { Button, ButtonKind } from "../../Styleguide/Button";
+import { Field } from "../../../backend/DB/models/field";
+import { Select } from "../../Styleguide/Select";
+
+export function EditIntakeTextField(props: EditItemProps) {
+  const intakeItem = props.intakeItem as IntakeFieldItem;
+  const [label, setLabel] = useState(intakeItem?.question.label ?? "");
+  const [required, setRequired] = useState(
+    intakeItem?.question.required ?? true
```

Comment:
The left side of this optional boolean can yield the value of null, true, or false.  This boolean will result in either true  or false.   "Nil coalescing operator"